Mu2eG4/inc/Mu2eG4Config.hh

```diff
@@ -200,9 +200,9 @@ namespace mu2e {
       fhicl::Atom<std::string> generatorModuleLabel {Name("generatorModuleLabel"), ""};
 
       fhicl::Atom<bool> G4InteralFiltering {Name("G4InteralFiltering"), false};
-      fhicl::Atom<int>  maxEventsToSeed {Name("maxEventsToSeed"),
-          Comment("Only used by Mu2eG4MT module."),
-          10000 };
+      fhicl::Atom<long> initialSeed {Name("initialSeed"),
+          Comment("Only used by Mu2eG4MTRunManager when running in MT mode"),
+          8 };
```

Comment:
See previous comment about using SeedService.

Reply:
I have removed the initialSeed FHICL parameter.  We are now using the Seed Service to provide the seed to the random number generator used by the Mu2eG4 master run manager.